ecosystem/sep-0001.md

```diff
@@ -6,8 +6,8 @@ Title: stellar.toml
 Author: stellar.org
 Status: Active
 Created: 2017-10-30
-Updated: 2021-04-20
-Version: 2.3.0
+Updated: 2021-04-29
+Version: 2.3.1
```

Comment:
Should bump minor version because it's adding new functionality and not giving existing functionality.